### PR TITLE
JupyterBook 0.13

### PR DESCRIPTION
**Pull request by scottyhq**
Addresses #32

I wanted to see how upgrading to 0.13 worked here. In general I think it will be easy to update hackweek jupyterbooks as well, but also happy to wait until after snowex 2022 is done.

I didn't realize how extensively "panels" where used in the guidebook, so this was more work than I expected, but I wanted to spend some time with the new formatting options. 

In general the new "grid" system is pretty nice and I like that the sidebar text default is smaller font compared to main document text.

I particularly like the way cards can become links that are highlighted when somebody mouses over them (for example the gallery page https://deploy-preview-33--hackweek-guidebook.netlify.app/gallery/index.html).

### CODE REVIEW

**Comment by scottyhq** [comment]
One of the major changes is how content is automatically re-arranged and sidebars automatically hidden based on web-browser window size.  See comparisons below for existing (0.12) on left and new (0.13) on right: 

Standard window (laptop split screen):

Screen Shot 2022-06-09 at 9 18 53 AM

Small window:

Screen Shot 2022-06-09 at 9 20 42 AM

File: docs/gallery/index.md [hunk 1]
```diff
@@ -1,36 +1,26 @@
...
 
----
-
-![ICESat-2 hackweek logo](http://www.nasa.gov/sites/default/files/thumbnails/image/icesat2-logo_0.jpg)
-
-```{link-button} ICESat-2hackweek
-:text: ICESat-2 Hackweek
-:type: ref
-:classes: stretched-link
+```{grid-item-card} ICESat-2 Hackweek 2020
```
Comment (JessicaS11) [inline docs/gallery/index.md:12]: Should we update to 2022?

File: docs/team.md [hunk 2]
```diff
@@ -1,113 +1,67 @@
...
-*Roles:* Education, administrative and technical specialist
+```{grid-item-card} Emilio Mayorga
+:img-top: https://i0.wp.com/communitysnowobs.org/wp-content/uploads/2020/12/mayorga.jpg?resize=300%2C300&ssl=1
+Education and technical specialist
+```
 
----
+```{grid-item-card} Jessica Scheick
+:img-top: https://avatars.githubusercontent.com/u/11756442?v=4 Education, administrative and technical specialist
+```
```
Comment (JessicaS11) [inline docs/team.md:55]:
```suggestion
:img-top: https://avatars.githubusercontent.com/u/11756442?v=4
Lead Coordinator, Education and technical specialist
```

```

**Review by JessicaS11 — CHANGES_REQUESTED** [review]

File: docs/gallery/index.md (hunk 1 above)
Comment (scottyhq) [reply docs/gallery/index.md:12]: It actually links to 2021 content only right now! We should probably just omit the date and have subsections for different events on https://guidebook.hackweek.io/gallery/SnowExhackweek.html ...

**Pull request merged**